http://hg.code.sf.net/p/graphicsmagick/code//rev/d9ec69e25e9e

Sent upstream.

# User Sam James <sam@gentoo.org>
# Date 1631846586 -3600
#      Fri Sep 17 03:43:06 2021 +0100
magick/blob.c: Fix build with bzip2 enabled and zlib disabled

Seems to be a typo.

Bug: https://bugs.gentoo.org/641164

--- a/magick/blob.c
+++ b/magick/blob.c
@@ -5040,7 +5040,7 @@ MagickExport size_t WriteBlob(Image *ima
             else
               amount=(int) remaining;
 
-            bz_count=BZ2_bzwrite(blob->handle.gz,
+            bz_count=BZ2_bzwrite(blob->handle.bz,
                                  (void *) ((unsigned char *) data+i),amount);
             if (bz_count <= 0)
               break;
